vs-dark/theme.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "VsDark",
+    "version": "1.0",
+    "description": "VsCode Dark Theme like for PhpMyAdmin",
+    "author": "Miumo",
+    "url": "https://github.com/MrMiumo",
+    "supports": ["5.0", "5.1", "5.2"]
```

Comment:
Are you sure about that ?
Did you test your theme ?
From what I see it should only be compatible with 4.9

Reply:
I really doubt this because you have .php files and this is long dead 🤔 
What theme did you use as an example ?

Here is an example of theme that uses the most modern theme methods: https://github.com/phpmyadmin/themes/tree/master/boodark

Reply:
Cool, maybe you did some upgrades without a cleanup of the old data :)
You can push a commit to remove them, I will continue my review right after

Reply:
Very cool, it looks better 💯 
Did you try your theme on the 3 different versions you advertise ?
If I recall correctly, it's not possible in a easy way to have compatibility for 5.0 and 5.2
Bootstrap versions ...

Reply:
> Do I only need to indicate the version(s) I tested the theme on or are there compatibility things?

The versions you indicate here will allow the theme to work on the stated phpMyAdmin versions, so it's pretty important.

You should only put the ones you tested as it quickly dives into specific changes for Bootstrap between versions

Reply:
@williamdes I want to create a theme for phpmyadmin, so what theme should I take as starter?

Reply:
The theme https://github.com/phpmyadmin/themes/tree/master/boodark is a very well maintained one
You can find more of the fine tooling here: https://github.com/adorade/boodark
I will leave it up to @adorade if they want to give some help on building themes using boodark as a template